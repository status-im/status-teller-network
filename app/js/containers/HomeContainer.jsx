import React, { Component } from 'react';
import PropTypes from 'prop-types';
import { Row, Col, Button } from 'reactstrap';
import { Link } from "react-router-dom";
import {withNamespaces} from 'react-i18next';

import "./HomeContainer.scss";

class HomeContainer extends Component {
  render() {
    const t = this.props.t;
    return (
      <div className="home">
        <Row className="home-headline">
          <Col xs={12}>
            <h1 className="text-center">{t('home.welcome')}</h1>
          </Col>
        </Row>

        <Row className="home--footer">
          <Col xs={6}>
            <Button tag={Link} color="info" block to="/buy">{t('home.buy')}</Button>
          </Col>
          <Col xs={6}>
            <Button tag={Link} color="info" block to="/sell">{t('home.sell')}</Button>
          </Col>
        </Row>
      </div>
    );
  }
}

HomeContainer.propTypes = {
  t: PropTypes.func
};

export default withNamespaces()(HomeContainer);
