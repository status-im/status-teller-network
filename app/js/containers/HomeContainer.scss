.home {
  .home-headline {
    margin-top: 17rem;
  }

  .home--footer {
    position: absolute;
    right: 0;
    bottom: 0;
    left: 0;
    height: 7rem;
  }
}