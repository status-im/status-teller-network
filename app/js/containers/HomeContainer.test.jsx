import React from "react";
import { shallow } from "enzyme";

import HomeContainer from "./HomeContainer";

describe('License', () => {
  it('should render correctly', () => {
    const component = shallow(<HomeContainer />);
  
    expect(component).toMatchSnapshot();
  });
});
