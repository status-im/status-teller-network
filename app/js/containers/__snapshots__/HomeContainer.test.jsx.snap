// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`License should render correctly 1`] = `
<div
  className="home"
>
  <Row
    className="home-headline"
    tag="div"
  >
    <Col
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
      xs={12}
    >
      <h1
        className="text-center"
      />
    </Col>
  </Row>
  <Row
    className="home--footer"
    tag="div"
  >
    <Col
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
      xs={6}
    >
      <Button
        block={true}
        color="info"
        tag={[Function]}
        to="/buy"
      />
    </Col>
    <Col
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
      xs={6}
    >
      <Button
        block={true}
        color="info"
        tag={[Function]}
        to="/sell"
      />
    </Col>
  </Row>
</div>
`;
